### PR TITLE
Designed the UI for userProfile

### PR DESCRIPTION
**Pull request by irenezhengg**

### CODE REVIEW

File: app/templates/userProfile.html [hunk 1]
```diff
@@ -1,13 +1,93 @@
...
+            </div>
+            <div class="col-md-8">
+              <div class="card-body p-4">
+                <h6>User Information</h6>
+                <hr class="mt-0 mb-4">
+                <div class="row pt-1">
+
+                  <div class="col-6 mb-3">
+                    <h6>First name</h6>
+                    <p class="text-muted">Irene</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:37]: please change to `{{ current_user.first_name }}`.

File: app/templates/userProfile.html [hunk 2]
```diff
@@ -1,13 +1,93 @@
...
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>Last name</h6>
+                    <p class="text-muted">Zheng</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:42]: please change to `{{ current_user.last_name }}`.

File: app/templates/userProfile.html [hunk 3]
```diff
@@ -1,13 +1,93 @@
...
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>email</h6>
+                    <p class="text-muted">hello@gmail.com</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:47]: please change to `{{ current_user.email }}`.

File: app/templates/userProfile.html [hunk 4]
```diff
@@ -1,13 +1,93 @@
...
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>username</h6>
+                    <p class="text-muted">adminone</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:52]: please change to `{{ current_user.username }}`.

File: app/templates/userProfile.html [hunk 5]
```diff
@@ -1,13 +1,93 @@
...
+                  <div class="col-6 mb-3">
+                    <h6>role</h6>
+                    <p class="text-muted">admin</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>last_login</h6>
+                    <p class="text-muted">active</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:65]: please change to `{{ current_user.last_login }}`.

File: app/templates/userProfile.html [hunk 6]
```diff
@@ -1,13 +1,93 @@
...
+
+                  <div class="col-6 mb-3">
+                    <h6>username</h6>
+                    <p class="text-muted">adminone</p>
+                  </div>
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>role</h6>
+                    <p class="text-muted">admin</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:58]: please change to `{{ current_user.role }}`.

File: app/templates/userProfile.html [hunk 7]
```diff
@@ -1,13 +1,93 @@
...
+                  <div class="col-6 mb-3">
+                    <h6>last_login</h6>
+                    <p class="text-muted">active</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>create_at</h6>
+                    <p class="text-muted">active</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:72]: please change to `{{ current_user.create_at }}`.

File: app/templates/userProfile.html [hunk 8]
```diff
@@ -1,13 +1,93 @@
...
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>modified_at</h6>
+                    <p class="text-muted">active</p>
```
Comment (Ming119) [inline app/templates/userProfile.html:79]: please change to `{{ current_user.modified_at }}`.

File: app/templates/userProfile.html [hunk 9]
```diff
@@ -1,13 +1,93 @@
...
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>last_login</h6>
+                    <p class="text-muted">active</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
```
Comment (Ming119) [inline app/templates/userProfile.html:69]: Those `div` tag for admin can share the `if statement`. To reduce the redundancy, please remove this.

File: app/templates/userProfile.html [hunk 10]
```diff
@@ -1,13 +1,93 @@
...
+                  <div class="col-6 mb-3">
+                    <h6>create_at</h6>
+                    <p class="text-muted">active</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
```
Comment (Ming119) [inline app/templates/userProfile.html:76]: Those `div` tag for admin can share the `if statement`. To reduce the redundancy, please remove this.

**Comment by irenezhengg** [comment]
The feedbacks above are resolved.

**Review by Ming119 — CHANGES_REQUESTED** [review]
@irenezhengg sorry, my mistake. Those `current_user` should be `user`

**Comment by irenezhengg** [comment]
> @irenezhengg sorry, my mistake. Those `current_user` should be `user`

done

**Review by irenezhengg — COMMENTED** [review]
done

**Review by irenezhengg — COMMENTED** [review]
done

File: app/templates/userProfile.html [hunk 11]
```diff
@@ -1,13 +1,78 @@
...
+        <div class="card mb-3" style="border-radius: .5rem;">
+          <div class="row g-0">
+            <div class="col-md-4 gradient-custom text-center text-white" style="border-top-left-radius: .5rem; border-bottom-left-radius: .5rem;">
+              <img
+                src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp"
+                alt="Avatar"
+                class="img-fluid my-5"
+                style="width: 80px;"
+              />
+              <h5>{{ current_user.username }}</h5>
```
Comment (Ming119) [inline app/templates/userProfile.html:17]: replace `current_user` to `user`.

**Review by Ming119 — CHANGES_REQUESTED** [review]

File: app/templates/userProfile.html (hunk 11 above)
Comment (irenezhengg) [reply app/templates/userProfile.html]: sorry, I forgot to push. the change has been done. 
image

**Review by Ming119 — APPROVED** [review]

**Review by HuangShengYao108590061 — APPROVED** [review]

**Pull request merged**